elasticsearch/src/elasticsearch/core.clj

```diff
@@ -175,20 +178,18 @@
   (setup! [_ test node]
     (let [; client (es/connect [[(name node) 9300]])]
           client (es/connect (str "http://" (name node) ":9200"))]
-      ; Create index
+      ;; Create index
       (try
         (esi/create client index-name
                     :mappings {"number" {:properties
                                          {:num {:type "integer"
                                                 :store "yes"}}}}
-                    :settings {"index" {"refresh_interval" "1"}})
-        (catch clojure.lang.ExceptionInfo e
-          ; Is this seriously how you're supposed to do idempotent
-          ; index creation? I've gotta be doing this wrong.
-          (let [err (http-error e)]
-            (when-not (re-find #"IndexAlreadyExistsException" err)
-              (throw (RuntimeException. err))))))
-
+                    :settings {"index" {"refresh_interval" "1s"}})
+        (catch Throwable t))
```

Comment:
I'd really prefer to know about connection errors etc that happen here; the only reason it's appropriate to noop is if the index already exists.